### PR TITLE
Feature/FP-34

### PR DESCRIPTION
**Pull request by natam**
FP-34 order models and repositories added

### CODE REVIEW

File: src/main/java/com/nkh/ECommerceShop/controller/OrdersController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    @Autowired
+    public OrdersController(OrdersRepository ordersRepository) {
+        this.ordersRepository = ordersRepository;
+    }
+
+    @GetMapping
+    @PreAuthorize("hasAnyRole('USER')")
+    public ResponseEntity<List<Order>> getAllOrders(){
+        return ResponseEntity.ok(ordersRepository.findAll());
```
Comment (MenilV) [inline src/main/java/com/nkh/ECommerceShop/controller/OrdersController.java:28]: what if there's a SQL exception here

**Review by MenilV — COMMENTED** [review]
On thing to check and we're good to go

File: src/main/java/com/nkh/ECommerceShop/controller/OrdersController.java (hunk 1 above)
Comment (MenilV) [inline src/main/java/com/nkh/ECommerceShop/controller/OrdersController.java:28]: what if `.findAll()` throws an `SQLException`. As much as I appreciate brevity in code and doing multiple things within

Comment (natam) [reply src/main/java/com/nkh/ECommerceShop/controller/OrdersController.java:28]: I pushed controller by mistake.This one is only for test purpose :) I used it to check my model

Comment (MenilV) [reply src/main/java/com/nkh/ECommerceShop/controller/OrdersController.java:28]: let's either remove it or fix it and then we can re-review

Comment (natam) [reply src/main/java/com/nkh/ECommerceShop/controller/OrdersController.java]: removed

**Review by MenilV — APPROVED** [review]

**Pull request merged**